apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateDateQuestion.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './CreateQuestion.scss';
+import { Radio } from '@trussworks/react-uswds';
+import { Control, Controller, FieldValues } from 'react-hook-form';
+import { SelectInput } from '../../../../components/FormInputs/SelectInput';
+type DateQuestionProps = {
+    control: Control<FieldValues, any>;
+};
+export const CreateDateQuestion = ({ control }: DateQuestionProps) => {
+    return (
+        <>
+            <Controller
+                control={control}
+                name="dateFormat"
+                rules={{ required: { value: true, message: 'Mask required' } }}
+                render={({ field: { onChange, value }, fieldState: { error } }) => (
```

Comment:
issue:  Validations should occur as the user id filling out the form.  The `useForm` is already set to a mode of `onBlur`, in order for the validations to trigger when the user tabs out if the field the `onBlur` handler should be passed to the controlled component.

` render={({ field: { onBlur, onChange, value }, fieldState: { error } }) => (`

Then you can pass the `onBlur` function from render into the component to get the desired functionality.